libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -75,7 +76,12 @@ export class CosmosAPI {
         withdrawAddress,
       };
     } catch (e) {
-      throw new Error(`"Error during cosmos synchronization: "${(e as Error).message}`);
+      const error = e as AxiosError;
```

Comment:
It's more like (AxiosError | Error) and then type narrowing should do the rest

Reply:
Nope doesn't work because `isAxios` or `toJSON` don't exist in type Error and we have to do extra casting

Reply:
Ok then :shrug: